cmake/scripts/windows/ArchSetup.cmake

```diff
@@ -1,9 +1,11 @@
-# Minimum SDK version we support
-set(VS_MINIMUM_SDK_VERSION 10.0.22621.0)
+# Minimum OS version to run the app
+set(VS_MINIMUM_SDK_VERSION 10.0.18362.0)
```

Comment:
This concept/feature of minimum OS version to run the app not exist on desktop applications, at least not exist the runtime check linked to this variable on VS studio project. Usually the installers should take care of this (NSIS in this case). Then I vote for remove double variable in desktop (as it can cause confusion) but maybe rename to `VS_TARGET_SDK_VERSION` (as it is in PR) to keep in sync with new variable name in UWP.

The variable `VS_MINIMUM_SDK_VERSION` only is used here:

https://github.com/xbmc/xbmc/blob/d0f011f789bb38f52849ce47203b198270335ec2/CMakeLists.txt#L526-L527

and effectively only is used in UWP. It works because value it's passed to CMake:

https://cmake.org/cmake/help/latest/prop_tgt/VS_WINDOWS_TARGET_PLATFORM_MIN_VERSION.html

Reply:
Sure. it doesn't hurt to not define the variable for Windows desktop, as it's used conditionally for Windows Store only in the CMakeLists.txt you pointed.